hasura/migrations/default/1663180519908_contributions_decouple_epochs/up.sql

```diff
@@ -0,0 +1,33 @@
+DROP table "public"."contributions";
+
+CREATE TABLE "public"."contributions"
+(
+  "id" bigserial NOT NULL,
+  "circle_id" bigint NOT NULL,
+  "user_id" bigint NOT NULL,
+  "description" text NOT NULL,
+  "datetime_created" timestamptz NOT NULL DEFAULT now(),
+  "created_at" timestamptz NOT NULL DEFAULT now(),
+  "updated_at" timestamptz NOT NULL DEFAULT now(),
+  "deleted_at" timestamptz,
+  PRIMARY KEY ("id"),
+  FOREIGN KEY ("circle_id") REFERENCES "public"."circles"("id") ON UPDATE restrict ON DELETE restrict,
+  FOREIGN KEY ("user_id") REFERENCES "public"."users"("id") ON UPDATE restrict ON DELETE restrict,
+  UNIQUE ("id")
+);
+CREATE OR REPLACE FUNCTION "public"."set_current_timestamp_updated_at"()
+RETURNS TRIGGER AS $$
```

Comment:
I think @levity and @CryptoGraffe didnt like this trigger in the other interaction_events PR, but not sure why not...